stubs/passlib/passlib/handlers/windows.pyi

```diff
@@ -17,8 +17,15 @@ class nthash(uh.StaticHandler):
     checksum_size: ClassVar[int]
     @classmethod
     def raw(cls, secret): ...
+    @overload
     @classmethod
-    def raw_nthash(cls, secret, hex: bool = False): ...
+    def raw_nthash(cls, secret: str | bytes, hex: Literal[True]) -> str: ...
+    @overload
+    @classmethod
+    def raw_nthash(cls, secret: str | bytes, hex: Literal[False]) -> bytes: ...
+    @overload
+    @classmethod
+    def raw_nthash(cls, secret: str | bytes, hex: bool = False) -> str | bytes: ...
```

Comment:
See my comments on a previous PR. I'd just drop the "default" overload, unless mypy_primer throws a fit.

Reply:
let's leave "default" overload